src/FactSystem/FactMetaData.cc

```diff
@@ -51,24 +56,31 @@ const FactMetaData::AppSettingsTranslation_s FactMetaData::_rgAppSettingsTransla
     { "m",      "m",        FactMetaData::UnitDistance,    UnitsSettings::DistanceUnitsMeters,         FactMetaData::_defaultTranslator,                   FactMetaData::_defaultTranslator },
     { "meter",  "meter",    FactMetaData::UnitDistance,    UnitsSettings::DistanceUnitsMeters,         FactMetaData::_defaultTranslator,                   FactMetaData::_defaultTranslator },
     { "meters", "meters",   FactMetaData::UnitDistance,    UnitsSettings::DistanceUnitsMeters,         FactMetaData::_defaultTranslator,                   FactMetaData::_defaultTranslator },
+    //NOTE: we've coined an artificial "raw unit" of "altitude metre" to separate it from the distance metre - a bit awkward but this is all the design permits
+    { "alt m",  "m",        FactMetaData::UnitAltitude,    UnitsSettings::AltitudeUnitsMeters,         FactMetaData::_defaultTranslator,                   FactMetaData::_defaultTranslator },
     { "cm/px",  "cm/px",    FactMetaData::UnitDistance,    UnitsSettings::DistanceUnitsMeters,         FactMetaData::_defaultTranslator,                   FactMetaData::_defaultTranslator },
     { "m/s",    "m/s",      FactMetaData::UnitSpeed,       UnitsSettings::SpeedUnitsMetersPerSecond,   FactMetaData::_defaultTranslator,                   FactMetaData::_defaultTranslator },
     { "C",      "C",        FactMetaData::UnitTemperature, UnitsSettings::TemperatureUnitsCelsius,     FactMetaData::_defaultTranslator,                   FactMetaData::_defaultTranslator },
-    { "m^2",    "m^2",      FactMetaData::UnitArea,        UnitsSettings::AreaUnitsSquareMeters,       FactMetaData::_defaultTranslator,                   FactMetaData::_defaultTranslator },
+    { "m^2",    "m\u00B2",  FactMetaData::UnitArea,        UnitsSettings::AreaUnitsSquareMeters,       FactMetaData::_defaultTranslator,                   FactMetaData::_defaultTranslator },
```

Comment:
while at it change to superscript - in case this isn't a favour, i will remove.

Reply:
@dogmaphobic Do you know anything about Qt Fonts. Will this cause problems?
@remster do you have screenshots of what this looks like?

Reply:
Is it possible somehow to have the table only be horizontal distance units. And then the vertical distance variants for all of these are generated automagically by prepending "vertical " to the front of the horizontal distance ones? There needs to be vertical variants for all the horizontal ones not just meters.

Reply:
> @remster do you have screenshots of what this looks like?
IS:
![superscript](https://user-images.githubusercontent.com/8327961/80545720-52883c00-89b4-11ea-80c5-1424486df758.png)
WAS:
... (coming up)

Reply:
Ah, super nice. Definitely want that if there any strange issues.

Reply:
So i understand what you want here:
> Is it possible somehow to have the table only be horizontal distance units. And then the vertical distance variants for all of these are generated automagically by prepending "vertical " to the front of the horizontal distance ones?

but do not here: 
> There needs to be vertical variants for all the horizontal ones not just meters.

Do you mean, on top of "m" also: "meter", "meters" - I don't understand why these raw extra units are there at all (neglecting the British spelling i might add) as "raw units" aren't (I may be wrong) presented to the user?
Or do you somehow also mean speed and area? Please just be more specific, I'll comply.

Reply:
> Ah, super nice. Definitely want that if there aren't any strange issues.

That I don't know how to go around verifying. Any hints how these cooked units are localised?

> There is on language which uses a different font, maybe Korean. Wonder if that will works as well. These are localized yet, but at some point we'll get around to supporting that.

I can do you Polish :)

Reply:
> Any hints how these cooked units are localised?

They aren't currently. If you look in QGCApplication.cc you'll see that Korean has a different font for some reason. Could you hack your build and verify that font works with these as well?

Reply:
There you go @DonLakeFlyer 
![korean](https://user-images.githubusercontent.com/8327961/80634540-ea406580-8a5a-11ea-9c51-128c58a7222a.png)

Reply:
Perfect. Seems fine then.